src/style.css

```diff
@@ -83,14 +83,22 @@
   content: attr(data-terminal-prompt);
 }
 
-.react-terminal-wrapper:focus-within .react-terminal-active-input:after {
+.react-terminal-wrapper:focus-within .react-terminal-active-input #cursor:after {
   content: '▋';
+  position: relative;
   font-family: monospace;
-  margin-left: 0.2em;
   -webkit-animation: blink 1s infinite;
           animation: blink 1s infinite;
 }
 
+/* .react-terminal-wrapper:focus-within .react-terminal-active-input:after {
+  content: '▋';
+  font-family: monospace;
+  margin-left: 0.2em;
+  -webkit-animation: blink 1s infinite;
+          animation: blink 1s infinite;
+} */
```

Comment:
Feel free to remove the commented out code if this is being replaced by the `#cursor:after` rule.

Reply:
Done